packages/ui/src/components/va-time-input/VaTimeInput.vue

```diff
@@ -77,8 +77,8 @@
             :tabindex="iconsTabIndexComputed"
```

Comment:
Pressing `Escape` should close the dropdown - identical to default `VaTimeInput` behavior.

Reply:
Fixed.